ConsoleUI/ModListScreen.cs

```diff
@@ -48,18 +48,17 @@ public ModListScreen(KSPManager mgr, bool dbg)
                 },
                 1, 0, ListSortDirection.Descending,
                 (CkanModule m, string filter) => {
+                    // Search for author
                     if (filter.StartsWith("@")) {
                         string authorFilt = filter.Substring(1);
                         if (string.IsNullOrEmpty(authorFilt)) {
                             return true;
-                        } else if (m.author != null) {
-                            foreach (string auth in m.author) {
-                                if (auth.IndexOf(authorFilt, StringComparison.CurrentCultureIgnoreCase) == 0) {
-                                    return true;
-                                }
-                            }
+                        } else {
+                            // Remove special characters from search term
+                            authorFilt = CkanModule.nonAlphaNums.Replace(authorFilt, "");
+                            return m.SearchableAuthors.IndexOf(authorFilt, StringComparison.CurrentCultureIgnoreCase) == 0;
                         }
-                        return false;
```

Comment:
I'm guessing that this `return false;` was there for a reason. Why was it removed?